OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -122,62 +132,116 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <desc>Mediocre hand with angular script. Word dividers are also
+                                    placed at the begining of a line (e.g. <locus
+                                        target="#5v #6r #10v"/>.</desc>
+                                <seg type="rubrication">Initial two lines on the incipit page of
+                                        <ref target="#ms_i1"/>; the first word of each stanza; name
+                                    of the protagonist Michael, erased names of individuals related
+                                    to the production or ownership of the manuscript; elements of
+                                    the punctuation marks, and numerals.</seg>
                             </handNote>
                         </handDesc>
-                        
-                       <!-- <decoDesc>
-                         
-                        </decoDesc>-->
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding" contemporary="false">
-                                <decoNote xml:id="b1">Modern binding</decoNote>
+
+                        <additions>
+                            <list>
+
+                                <item xml:id="e1">
+                                    <desc>Scribal corrections appear interlineally (e.g. <locus
+                                            target="#3v #4r #4v"/>) or over erasure comprising
+                                        several words (e.g. <locus target="#12r #13v"/>).</desc>
+                                </item>
+
+                                <item xml:id="e2">
+                                    <desc>Writing exercises and crude drawings on <locus from="1r"
+                                            to="1v"/> and <locus from="15v" to="16v"/>; scribbles
+                                        and doodles in the margins throughout the manuscript;
+                                        erased notes in brown ink in the lower margin of <locus
+                                            from="9r" to="11r"/>.</desc>
+                                </item>
+
+                                <item xml:id="e3">
+                                    <desc> A white sticker with the name of <persName
+                                            ref="PRS5782JuelJen"/> printed (<foreign xml:lang="gez"
+                                            >ቤንት፡ ዩውል፡ የንሰን።</foreign>), and the signatures Ms.
+                                        Aeth. g. 28 and Ms. Et 1 written in pencil are glued onto
+                                        the inner side of the front board. Below, another sticker
+                                        with the signature ‘Ms 1’ printed on it is glued. A note is
+                                        written in pencil in the inner side of the front and back
```

Comment:
A white sticker .... IS glued. Another sticker with the signature ‘Ms 1’ is glued below it.

p.s.
(Can we omit that it's printed? It seems like too much info on stickers to me!)

Reply:
We might keep this information for the benefit of the online database (where any manuscript should be considered a single individual object by potential users), and decide not to print that information in our catalogue. I do not know exactly how to do this, but in my view it is better asking for not printing something we deem as redundant, rather than omitting it in the XML description.

Reply:
This is exactly what I meant. Thanky for clarifying it!